Sources/MapLibreSwiftUI/MapView.swift

```diff
@@ -103,13 +103,16 @@ public struct MapView<T: MapViewHostViewController>: UIViewControllerRepresentab
 
         if cameraDisabled == false {
             context.coordinator.updateCamera(mapView: uiViewController.mapView,
-                                             camera: $camera.wrappedValue,
+                                             camera: camera,
```

Comment:
I sometimes forget the details of binding syntactic sugar, but I am pretty sure this is equivalent.